doc/classes/Basis.xml

```diff
@@ -148,13 +181,33 @@
 			<param index="1" name="angle" type="float" />
 			<description>
 				Introduce an additional rotation around the given axis by [param angle] (in radians). The axis must be a normalized vector.
+				[codeblock]
+				var my_basis = Basis.IDENTITY
+				var angle = PI / 2
+
+				my_basis = my_basis.rotated(Vector3.UP, angle)      # Rotate around the up axis (yaw)
+				my_basis = my_basis.rotated(Vector3.RIGHT, angle)   # Rotate around the right axis (pitch)
+				my_basis = my_basis.rotated(Vector3.FORWARD, angle) # Rotate around the forward axis (roll)
```

Comment:
The +Z axis is Vector3.BACK. If you rotate around Vector3.FORWARD then the rotation would be the reverse of the usual direction people expect to rotate in a right-handed coordinate system (different from Euler angles of `(0, 0, angle)`)

```suggestion
				my_basis = my_basis.rotated(Vector3.BACK, angle) # Rotate around the back axis (roll)
```